lesson_9/alex_roman/lesson_9_roman_alex.py

```diff
@@ -0,0 +1,65 @@
+# Build two classes of your choice that can model a real-life example. The class needs to meet the following requirements:
+
+    # at least 5 attributes each
+    # at least 2 methods each
+    # one class to inherit from another
+
+# As a demonstration create at least 5 instances of one class (preferably the child class) and call all the methods it holds
+
+class Client:
+    no_of_clients = 0
+
+    def __init__(self, first_name, last_name, country, telephone, company):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.country = country
+        self.telephone = telephone
+        self.company = company
+        Client.no_of_clients += 1
```

Comment:
this is a nice usage of class attributes. This will affect the value of no_of_clients for the instances: 
first instance -> no_of_clients = 1
secoud instance -> no_of_clients = 2 

If this is the desired behaviour everything is fine :smile: